backend/events/apps.py

```diff
@@ -192,33 +195,37 @@ def wait():
 
                     channel_event_hub.registerChaincodeEvent(chaincode_name,
                                                              'chaincode-updates',
-                                                             onEvent=on_event)
+                                                             onEvent=on_channel_event)
 
                     logger.info('Connect to Channel Event Hub')
+                    logger.info(f'Connect to Channel Event Hub ({channel_name})')
                     loop.run_until_complete(stream)
 
                 except RpcError as e:
-                    logger.error(f'Channel Event Hub failed ({type(e)}): {e} re-connecting in 5s')
+                    logger.error(f'Channel Event Hub failed for {channel_name} ({type(e)}): {e} re-connecting in 5s')
                     time.sleep(5)
 
 
 class EventsConfig(AppConfig):
     name = 'events'
 
-    def ready(self):
-
+    def listen_to_channel(self, channel_name):
         # We try to connect a client first, if it fails the backend will not start
         # It avoid potential issue when we launch the channel event hub in a subprocess
         while True:
             try:
-                with get_hfc() as (loop, client):
-                    logger.info('Start the event application.')
+                with get_hfc(channel_name) as (loop, client):
+                    logger.info(f'Events: Connected to channel {channel_name}.')
             except Exception as e:
                 logger.exception(e)
                 time.sleep(5)
-                logger.error('Retry to connect the event application to the ledger')
+                logger.error(f'Events: Retry connecting to channel {channel_name}.')
             else:
                 break
 
-        p1 = multiprocessing.Process(target=wait)
+        p1 = multiprocessing.Process(target=wait, args=[channel_name])
```

Comment:
It could be nice to test `Process` with `daemon=True` option (https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.daemon)

Reply:
I don't know if it will work as the backend is launch in a process by uwsgi !

Reply:
I'd argue we should keep this out of scope for this PR